Lombiq.Tests.UI/Services/TimeoutConfiguration.cs

```diff
@@ -27,13 +35,27 @@ public class TimeoutConfiguration
     /// </summary>
     public TimeSpan PageLoadTimeout { get; set; }
 
-    public static readonly TimeoutConfiguration Default = new()
+    /// <summary>
+    /// Gets or sets how long an individual test can run to prevent hanging indefinitely. Defaults to 600s.
```

Comment:
```suggestion
    /// Gets or sets how long an individual test can run to prevent hanging indefinitely. Defaults to 10 minutes.
```

Reply:
I'd prefer to keep the unit of the "Defaults" the same as the unit of the property.

Reply:
The config key is `TimeoutConfiguration:TestRunTimeoutSeconds`.

Reply:
I don't see how that's relevant in the docs of this property. As a developer interacting with this property, you don't need to care about a possible backing config key.

Reply:
If you configure it in one place using seconds but document it in other place using minutes, it's just needless inconsistency. But if you insist...